inverse_kinematics_node:
  ros__parameters:
    # Topics and frames
    target_topic: "/teleop/ee_target"
    joint_target_topic: "/joint_states"
    mycobot_base_frame: "mycobot_base"
    ee_child_frame: "gripper_ee"

    # IK tuning parameters
    position_tolerance: 0.003       # meters
    orientation_tolerance: 0.005    # radians
    max_iterations: 250
    damping: 0.001
    step_tolerance: 2.0e-6          # rad magnitude of joint update
    gripper_percent: 100            # default gripper position in percent
